account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -81,7 +88,7 @@ def _prepare_move_line_domain(self):
         elif self.date_type == "move":
             domain.append(("date", "<=", self.move_date))
         if self.invoice:
-            domain.append(("invoice_id", "!=", False))
+            domain.append(("move_id", "!=", False))
```

Comment:
This domain is a nonsense as is, as `move_id` is mandatory. I think you should put `("move_id.type", "in", ...)`

Reply:
Changed the domain for invoice!